cmd/worker/worker.go

```diff
@@ -354,6 +360,20 @@ func (c *Command) addClickhouseColumns(ctx context.Context, fields []*tunnel.Eve
 	return nil
 }
 
+func (c *Command) isEventsTableEmpty(ctx context.Context) (bool, error) {
+	var columns uint64
+	if err := c.clickhouse.QueryRow(ctx, `
+		SELECT COUNT(*)
+		FROM INFORMATION_SCHEMA.COLUMNS
+		WHERE (table_name = 'events') AND (table_schema = 'subtrace')
+	`).Scan(&columns); err != nil {
+		return false, fmt.Errorf("SELECT COUNT(*): %w", err)
+	}
+	// We expect only the insert_time column to be present when the table is initially created,
+	// see 2024-05-27-events.sql
```

Comment:
Not a huge fan of relying on the schema this way but I think it should be fine for now

Reply:
Maybe we could filter columns by `/^T([0-9a-fA-F]){8}$/` to see if there are any "event" columns in the table?

Reply:
Nah, this is fine, we can just change this function if that invariant changes